signaling/room_test.go

```diff
@@ -73,4 +77,163 @@ var _ = Describe("Room", func() {
 			Expect(message.Type).To(Equal(signaling.MessageTypeLeave))
 		})
 	})
+
+	Context("GetMember", func() {
+		var (
+			room          *signaling.Room
+			fakeMember    *signalingfakes.FakeRoomMember
+			anotherMember *signalingfakes.FakeRoomMember
+		)
+
+		BeforeEach(func() {
+			room = signaling.NewRoom()
+
+			fakeMember = &signalingfakes.FakeRoomMember{}
+			fakeMember.IDReturns(signaling.PeerID("123"))
+			room.AddMember(fakeMember)
+		})
```

Comment:
Please move this to Describe's block, which will allow it to be removed from all the Context blocks.